packages/schema-generator/src/generate_schema.test.ts

```diff
@@ -54,22 +54,60 @@ void describe('SchemaGenerator', () => {
     assert.strictEqual(dbConfig.port, 3306);
   });
 
-  void it('should throw error if one or more parts are missing in the database url', async () => {
-    const parse = () => parseDatabaseUrl('mysql://hostname/db');
-    assert.throws(parse, {
-      name: 'DatabaseUrlParseError',
-      message:
-        'Unable to parse the database URL. One or more parts of the database URL is missing. Missing [username, password, port].',
-    });
```

Comment:
This test still exists. I only removed validation for `port` as this will now use the default port number for a given engine if not provided.
https://github.com/aws-amplify/amplify-backend/pull/1256/files#diff-cb2f7d282d3892f824dd65fb652d391e557626eacbe08d92687d07a85fecac3aR105